app/stac_api/s3_multipart_upload.py

```diff
@@ -146,74 +131,32 @@ def complete_multipart_upload(self, key, asset, parts, upload_id):
         Raises:
             ValidationError: when the parts are not valid
```

Comment:
or ValueError when multipart upload fails

Reply:
@hansmannj well I didn't document the ValueError() because it is not clear to me when it happens. From the AWS docu it can happen but don't know in which case and/or circumstances. I couldn't force such issues from a user perspective. So I let this exception happen with a log error, and if it would happen the service would simply return an HTTP 500 with hopefully enough logs and information to understand why this happens. Only then we could want to change the logic to return eventually a 400 or any other more appropriate error code. For the moment I think it is better to have a 500 as we don't know in which circumstance it could happen.